.github/actions/run-in-docker-action/action.yml

```diff
@@ -37,6 +37,7 @@ runs:
         cache-from: type=gha
 
     - # Workaround for https://github.com/google/sanitizers/issues/1614 .
+      # The underlying issue has been fixed in clang 8.1.3.
```

Comment:
s/8.1.3/18.1.3/

Reply:
Amazing that I got this one-line commit wrong. Fixed.